torchtune/modules/peft/lora.py

```diff
@@ -9,7 +9,11 @@
 import torch.nn.functional as F
 
 from torch import nn, Tensor
-
+from torchao.dtypes.nf4tensor import linear_nf4
+from torchtune.modules.low_precision import (  # noqa: F401
+    _register_nf4_dispatch_ops,
```

Comment:
Trying to understand how this works: so basically we import `_register_nf4_dispatch_ops` at the first import of lora.py, which will then ensure any NF4Tensor used henceforth in the same run will have all those overrides in the ops table? Assuming that is the correct understanding, is there a way to do this with less of a hammer? (I don't think it matters for this particular PR, just curious if we try to extend things down the line)

Reply:
Yep, when this is imported these ops get registered on the NF4Tensor. 

>  is there a way to do this with less of a hammer

Hmm, not sure what would be less of a hammer - I thought this was already pretty light in that we just handle importing something and the registration automatically happens, we don't have to call any specific `register_ops` API or anything like that. Did you have any example of what you'd like this to be?